### PR TITLE
New website export

### PR DESCRIPTION
**Pull request by austinhinderer**
Copy of the new website found at the staging link

### CODE REVIEW

**Comment by canesin** [comment]
@CityOfZion/council take a look at this.. 💯

**Comment by Ejhfast** [comment]
is this the website on staging link here: http://fkmelrmflemrfelrkfmkermkfjn3n.s3-website-us-east-1.amazonaws.com ? 

looks great!

**Comment by AshRolls** [comment]
looks good.

1. docs link in projects is 404
2. neonwallet.com is outdated (links to 0.0.4 exe)

**Comment by M-L-A** [comment]
Seems like security review section was removed?

**Comment by canesin** [comment]
1 - Missing http://monitor.cityofzion.io project.
2 - Links to Facebook, Twitter and Medium
3 - Standardize to CoZ .. we have voted and are phasing out COZ.

**Comment by austinhinderer** [comment]
@AshRolls Will fix links.
@anfn101 Was directed to remove this section as there was no page to link to yet.
@canesin I will update the CoZ copy in the content. For the other two, are they just content suggestions or is there something missing from the mockups that I need to add? If not we need to circle back with hamster and get fresh assets.

**Comment by austinhinderer** [comment]
@AshRolls I pulled that link off of the current website. Is there a correct target this should point at? Also, that link for neonwallet came from the live site. Is there a better link I should use?

**Comment by austinhinderer** [comment]
Posted some responsive updates from hamster, as well as fixed a few instances of COZ -> CoZ.

- I need a place to link to for the documentation
- We'll follow up with security review once the content is ready
- Hamster is going to follow up with me on monitor.cityofzion.io

Hamster has approved the current staging link from a design perspective. Let me know if you see any problems with what we have, otherwise we should push out what we have and begin iterative improvements (after updating the broken docs link).

**Comment by M-L-A** [comment]
You can link here for docs: https://github.com/neo-project/docs

**Comment by austinhinderer** [comment]
Updated the link

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+<<<<<<< HEAD
```
Comment (canesin) [inline manifest.json]: Missing conflict resolution here

**Review by canesin — CHANGES_REQUESTED** [review]

**Comment by Nat1988** [comment]
We need copy for the NEO Monitor - can anyone provide this please?

**Comment by canesin** [comment]
@Nat1988 https://github.com/CityOfZion/neo-mon ?

**Comment by Nat1988** [comment]
Perfect!

File: manifest.json (hunk 1 above)
Comment (austinhinderer) [reply manifest.json]: Thanks, removed.

**Comment by austinhinderer** [comment]
- Added a link to NeoMon
- Added social media links
- Corrected merge conflict

**Review by canesin — APPROVED** [review]

**Comment by canesin** [comment]
I merged so we can move forward.. but where is the sources for the uncompiled website ? do you have instructions on how to edit ? ..
Also the slack link should be https://slack.cityofzion.io

**Comment by austinhinderer** [comment]
The source is in a branch on the Visual Identity repo. I suggest we break it into its own repo for easy CI in the future.

I put together a PR with the change for the slack link: https://github.com/CityOfZion/cityofzion.github.io/pull/19

**Pull request merged**